app/Console/Commands/UserCreate.php

```diff
@@ -101,4 +133,33 @@ public function handle(DiscourseService $discourseService): void
             $user->generateAndSetUsername();
         }
     }
+
+    /**
+     * Get the role name from the role ID.
+     *
+     * @param int $roleId
+     * @return string
+     */
+    private function getRoleName($roleId)
+    {
+        $roles = [
+            Role::ROOT => 'ROOT',
+            Role::ADMINISTRATOR => 'ADMINISTRATOR',
+            Role::HOST => 'HOST',
+            Role::RESTARTER => 'RESTARTER',
+            Role::GUSET => 'GUEST',
+            Role::NETWORK_COORDINATOR => 'NETWORK_COORDINATOR',
+        ];
+
+        return $roles[$roleId] ?? 'Unknown Role';
+    }
+
+    /**
+     * Validate a date string against a format.
+     */
+    private function validateDate($date, $format = 'Y-m-d')
+    {
+        $d = \DateTime::createFromFormat($format, $date);
```

Comment:
[super minor] We never pass in a `format`, so it doesn't really need to be a param.

```suggestion
     * Validate a date string for proper format.
     */
    private function validateDate($date)
    {
        $format = 'Y-m-d';
        $d = \DateTime::createFromFormat($format, $date);
```

Reply:
Addressed in [f371896](https://github.com/iFixit/restarters/pull/15/commits/f371896246dac0eac6853fee70b487abcaaf1bab)